### ISSUE TITLE
[Analytical Table]: [Accessibility issue - Not able to use keyboard tab to view(read) header and row cell]

### ISSUE THREAD

**Issue by JackieWei** [SAP/ui5-webcomponents-react#7058]
### Describe the bug

Open screen reader, try use keyboard tab key to navigate into table, only first column of header can be read out, for other cell, columm, row, all set tabindex=-1 which prevent user to read the table content by screen reader.

### Reproduction steps

1. Open screen reader by command + F5
2. Use tab key navigate to Analytical table instane 
3. Try to read the content
4. Enter into first cell, it will read content
5. Then tab to next, it will leave table
...

### Expected Behaviour

he screen reader should announce:
• The table role.
• Column headers (e.g.,).
• Row data corresponding to the selected cell.

### UI5 Web Components for React Version

Ui5  2.5.0

### UI5 Web Components Version

Ui5  2.5.0

### Browser

Chrome

### Operating System

Mac

### Relevant log output

```shell

```

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Hi @JackieWei 

please state the Screen Reader you're using. For some you may have to switch modes (e.g. switching off Virtual Cursor mode (Ins + Z) in JAWS).

As all complex ui5 components, the `AnalyticalTable` implements keyboard navigation. Therefore, you can navigate (i.a.) via arrow keys. Pressing Tab will either focus the next interactive element inside a cell or the next interactive element outside the table. This is by design.

**Comment by JackieWei** [comment]
@Lukas742  Thanks for answering. And will get back to accessibility colleagues to check first.
